stories/Iconography.stories.jsx

```diff
@@ -38,13 +55,14 @@ export const Iconography = () => {
           <span>Source</span>
         </a>
       </p>
-      <div className="grid grid-cols-4 lg:grid-cols-8 gap-3">
+      <div className="grid grid-cols-4 gap-3 lg:grid-cols-8">
+        {console.log(iconset.iconList)}
```

Comment:
Please remove this console statement here.

Reply:
> Please remove this console statement here.

Done.